Makefile

```diff
@@ -12,6 +12,8 @@ WHITE        := $(shell tput -Txterm setaf 7)
 
 RESET := $(shell tput -Txterm sgr0)
 
+L10N_MODULES := ./accounts ./settings
```

Comment:
get em programtically

Reply:
try this: `echo ./foo/baz/test | awk -F"/" '{print $0}'`

Reply:
does this work on your machine? 

![image](https://user-images.githubusercontent.com/26169327/112438496-446ba880-8d48-11eb-9950-f64a3ccfacf3.png)

Reply:
oh... this is zsh specific :-(